app/controllers/api/teams.php

```diff
@@ -627,6 +627,10 @@
             throw new Exception('Team IDs don\'t match', 404);
         }
 
+        if ($membership->getAttribute('confirm') !== false) {
+            throw new Exception('Membership already confirmed', 400);
```

Comment:
```suggestion
            throw new Exception('Membership already confirmed', 409);
```

I think 409 will be more descriptive here.

Reply:
Reference: https://appwrite.io/docs/error-codes